File: rules-interpreter/src/main/java/io/github/iamrenny/ruleflow/visitors/GrammarVisitor.java
package io.github.iamrenny.ruleflow.visitors;


import io.github.iamrenny.ruleflow.RuleFlowLanguageBaseVisitor;
import io.github.iamrenny.ruleflow.RuleFlowLanguageParser;

public class GrammarVisitor extends RuleFlowLanguageBaseVisitor<String> {

    @Override
    public String visitParse(RuleFlowLanguageParser.ParseContext ctx) {
        return removeSingleQuote(ctx.workflow().workflow_name().getText());
    }

    private String removeSingleQuote(String text) {
        return text.replace("'", "");
    }
}

File: rules-interpreter/src/main/kotlin/com/rappi/analang/visitors/GrammarVisitor.kt
package com.github.iamrenny.rulesflow.visitors

import com.rappi.fraud.analang.ANABaseVisitor
import com.rappi.fraud.analang.ANAParser
import com.github.iamrenny.rulesflow.removeSingleQuote

class GrammarVisitor : ANABaseVisitor<String>() {

    override fun visitParse(ctx: ANAParser.ParseContext): String {
        return ctx.workflow().workflow_name().text.removeSingleQuote()
    }
}